src/main/java/com/club_board/club_board_server/dto/myPage/userInfo/UpdateUserInfoRequest.java

```diff
@@ -17,23 +18,23 @@ public class UpdateMyPageRequest {
     @NotBlank
     private String department;
 
-    @NotBlank
+    @NotNull
     private int grade;
```

Comment:
```suggestion
    @NotNull
    private Integer grade;
```

원시 타입을 쓰게 되면 null 값이 존재하지 않기 때문에 해당 `@NotNull`이 무용지물이 될 것 같습니다. 게다가 임의의 기본 값이 들어갈 수 있어서 원시 타입보다는 래퍼 클래스로 사용하는 것이 어떨까요?

Reply:
오 좋은거같아요 바로 적용하겠습니다